src/lib/arch/aarch64/ExceptionHandler.cc

```diff
@@ -615,15 +613,16 @@ bool ExceptionHandler::init() {
         uint64_t bufPtr = registerFileSet.get(R0).get<uint64_t>();
         size_t buflen = registerFileSet.get(R1).get<size_t>();
 
-        char buf[buflen];
+        std::vector<char> buf;
```

Comment:
I don't mind either way. Will update to uint8_t for consistency

Reply:
Reverted back to char as otherwise this requires a reinterpret cast when instantiating the RegisterValue. If we want all chars to be uint8_t then this is a wider change and not part of this PR